testsuite/features/profiles/Kiwi/POS_Image-JeOS7_head/config.xml

```diff
@@ -94,12 +95,10 @@
         <package name="openssh"/>
         <package name="rsync"/>
         <package name="plymouth-plugin-label-ft"/>
-
         <package name="kernel-default"/>
-	<package name="salt-minion"/>
```

Comment:
if change in the spaces is unintentional, please revert

if intentional, thanks